builtin/mainmenu/settings/dlg_settings.lua

```diff
@@ -516,5 +516,9 @@ end
 
 
 function create_settings_dlg()
-	return dialog_create("dlg_settings", get_formspec, buttonhandler, nil)
+	local dlg = dialog_create("dlg_settings", get_formspec, buttonhandler, nil)
```

Comment:
i think it's better to have this in the dialog_create function, as it should be for all dialog, not only settings

Reply:
?